packages/components/psammead-story-promo/src/IndexAlsos/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import { node, string, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { C_EBON, C_METAL, C_LUNAR } from '@bbc/psammead-styles/colours';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const StyledIndexAlsos = styled.div`
+  position: relative;
+  z-index: 2;
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+    visibility: hidden;
+  }
+`;
+
+const StyledIndexAlso = styled.div`
+  border-top: 1px solid ${C_LUNAR};
+  padding: ${GEL_SPACING} 0;
+`;
+
+const StyledIndexAlsosUl = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+`;
+
+const StyledIndexAlsosLi = styled.li`
```

Comment:
If this is really supposed to have the same css as `StyledIndexAlso` you could replace by using `<StyledIndexAlso as="li" role="listitem">`. Not a big deal either way though